certora/helpers/MerkleTrees.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./MerkleTreeLib.sol";
+
+contract MerkleTrees {
+    using MerkleTreeLib for MerkleTreeLib.Node;
+    using MerkleTreeLib for MerkleTreeLib.Tree;
+
+    mapping(address => MerkleTreeLib.Tree) internal trees;
+
+    function newLeaf(address tree, MerkleTreeLib.Leaf memory leaf) public {
+        trees[tree].newLeaf(leaf);
+    }
+
+    function newInternalNode(address tree, MerkleTreeLib.InternalNode memory internalNode) public {
+        trees[tree].newInternalNode(internalNode);
+    }
+
+    function getLeft(address tree, address node) public view returns (address) {
+        return trees[tree].nodes[node].left;
+    }
+
+    function getRight(address tree, address node) public view returns (address) {
+        return trees[tree].nodes[node].right;
+    }
+
+    function getValue(address tree, address node) public view returns (uint256) {
+        return trees[tree].nodes[node].value;
+    }
+
+    function getHash(address tree, address node) public view returns (bytes32) {
+        return trees[tree].nodes[node].hashNode;
+    }
+
+    function isEmpty(address tree, address node) public view returns (bool) {
+        return trees[tree].nodes[node].isEmpty();
+    }
+
+    function isWellFormed(address tree, address node) public view returns (bool) {
+        return trees[tree].isWellFormed(node);
+    }
+
+    // Check that the nodes are well formed on the path from the root.
+    function wellFormedPath(
+        address tree,
+        address node,
```

Comment:
Hm yes, not sure how to write this comment because there is no notion of root anymore in this file. Here the only thing that matters is that it only reverts when `isWellFormed` is false on some node

Reply:
Added something that describes the argument at a high level